libraries/lib-module-manager/PluginManager.cpp

```diff
@@ -1143,6 +1143,19 @@ ComponentInterface *PluginManager::Load(const PluginID & ID)
    return nullptr;
 }
 
+void PluginManager::ClearEffectPlugins()
+{
+   for (auto it = mRegisteredPlugins.cbegin(); it != mRegisteredPlugins.cend(); ++it)
+   {
+      auto& desc = it->second;
+      if (desc.GetPluginType() == PluginTypeEffect)
+      {
+         mLoadedInterfaces.erase(desc.GetID());
+         it = mRegisteredPlugins.erase(it);
```

Comment:
Incorrect loop.  Increment the iterator or erase it, not both.  Or else you skip some elements.  May crash if you erase the last element.

Reply:
Done.  Moved iterator increment to the body of the for loop, and when not erasing.